load.php

```diff
@@ -291,6 +291,17 @@ function perflab_maybe_set_object_cache_dropin() {
 		return;
 	}
 
+	/**
+	 * Filters whether the Perflab server timing drop-in should be set.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param bool Whether the server timing drop-in should be set.
+	 */
+	if ( apply_filters( 'perflab_disable_object_cache_dropin', false ) ) {
```

Comment:
For clarity of intent, I'd suggest `enable` with a default of `true` rather than `disable` with a default of `false`.

Reply:
Even tho a few lines up it says `PERFLAB_DISABLE_OBJECT_CACHE_DROPIN`?

Reply:
Sorry, missed that as it's outside the diff. Let's go with the awkward phrasing.